backend/clubs/views.py

```diff
@@ -1230,6 +1230,38 @@ def post(self, request, *args, **kwargs):
         )
 
 
+class LastEmailInviteTestAPIView(APIView):
+    """
+    get: Return the club code, invite id and token of the last sent email invite
+    """
+
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request):
+        latest_email_invite = MembershipInvite.objects.filter(active=True).latest("created_at")
+        club_code = latest_email_invite.club.code
+        email_id = latest_email_invite.id
+        email_token = latest_email_invite.token
+
+        if request.user.email == latest_email_invite.email:
+            return Response(
+                {
+                    "code": club_code,
+                    "id": "{}".format(email_id),
```

Comment:
You can remove the `"{}".format` from these too, they're both strings. If you want to know what the type is of a field, you can check in `models.py`.